### PR TITLE
#5989 - feat: footer social component created.

### PR DESCRIPTION
**Pull request by DipakShrestha18**

### CODE REVIEW

File: src/components/Footer/FooterSocial/FooterSocial.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import FooterSocialItem from './FooterSocialItem';
+
+type FooterSocialProps = {
+  socialLinks: [];
+};
+
+export const FooterSocial = ({ socialLinks }: FooterSocialProps) => {
```
Comment (chriddell) [inline src/components/Footer/FooterSocial/FooterSocial.tsx]: Rename `socialLinks` props to `links` - "social" not needed as we're already inside the component.

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  text: string;
+  url: string;
+};
+
+export const FooterSocialItem = ({
+  className,
+  text,
+  url
+}: FooterSocialItemProps) => {
+  const classNames = cx('footer-social__icon newwindow-active', {
```
Comment (chriddell) [inline src/components/Footer/FooterSocial/FooterSocialItem.tsx]: Can we remove the `newwindow-active` classname for now? And we'll re-implement in a separate PR (giving us scope to refactor how this is done)

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    [`${className}`]: className
```
Comment (chriddell) [inline src/components/Footer/FooterSocial/FooterSocialItem.tsx]: also remove the `className` prop to keep things as clean as possible for now

**Review by chriddell — CHANGES_REQUESTED** [review]

**Comment by DipakShrestha18** [comment]
@chriddell re-review pls.

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cx from 'classnames';
```
Comment (chriddell) [inline src/components/Footer/FooterSocial/FooterSocialItem.tsx]: Can remove this import now (not using classnames any more)

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+type FooterSocialItemProps = {
+  text: string;
+  url: string;
+};
+
+export const FooterSocialItem = ({ text, url }: FooterSocialItemProps) => {
+  return (
+    <a href={url} rel="noopener noreferrer" target="_blank">
+      <span className="visuallyhidden">Wellcome on {text}</span>
```
Comment (chriddell) [inline src/components/Footer/FooterSocial/FooterSocialItem.tsx]: `visuallyhidden` should be `u-visually-hidden` (see https://github.com/wellcometrust/corporate-components/blob/develop/src/assets/styles/50-utilities/_a11y.scss)

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+      <span className="assistive-text"> (opens in a new tab)</span>
```
Comment (chriddell) [inline src/components/Footer/FooterSocial/FooterSocialItem.tsx]: Let's remove the `assistive text` className for now as it's not doing anything - we'll re-implement later

**Review by chriddell — APPROVED** [review]

**Review by chriddell — CHANGES_REQUESTED** [review]
update to request changes (accidentally hit _Approve_ on my last review!)

**Pull request merged**